GameOfRunes/GameOfRunes/Custom Nodes/Game Map/Stage Selection/Avatar Overview/PowerUpDescriptionNode.swift

```diff
@@ -35,8 +33,11 @@ class PowerUpDescriptionNode: SKSpriteNode, SelectedPowerUpResponder {
         fatalError("init(coder:) has not been implemented")
     }
     
-    func selectedPowerUpDidChanged(oldValue: PowerUpType?, newSelectedPowerUp: PowerUpType?) {
-        var description = newSelectedPowerUp?.description.components(separatedBy: "\n") ?? ["", "", ""]
+    func updatePowerUpDescription() {
+        var description = powerUpContainerNode
+            .selectedPowerUp?
+            .description
+            .components(separatedBy: "\n") ?? ["", "", ""]
         while description.count < 3 {
```

Comment:
is this considered magic number hahaha

Reply:
yeah i guess so, its how the description is constructed

Reply:
Yes hahaha. Because the description node can only support up to 3 lines of text. SpriteKit doesn't have multiline labels/textfields, so if we need `n` lines of text, need to ownself build it which I think not worth uh for 1 time use.